crates/ruff_python_semantic/src/analyze/typing.rs

```diff
@@ -965,6 +987,11 @@ pub fn is_type_var_like(binding: &Binding, semantic: &SemanticModel) -> bool {
     check_type::<TypeVarLikeChecker>(binding, semantic)
 }
 
+/// Test whether the given binding is for a `NewType`.
+pub fn is_newtype(binding: &Binding, semantic: &SemanticModel) -> bool {
```

Comment:
Is this used anywhere?

Reply:
I prefer only adding code that is used somewhere.

Reply:
Somehow I missed looking at this file while reviewing 🤦